lib/lndclient/LndClient.ts

```diff
@@ -506,20 +506,28 @@ class LndClient extends SwapClient {
   private executeSendRequest = async (
     request: lndrpc.SendRequest,
   ): Promise<string> => {
+    if (!this.isConnected()) {
+      throw swapErrors.FINAL_PAYMENT_ERROR(errors.UNAVAILABLE(this.currency, this.status).message);
```

Comment:
Why is this a final error? 
We are clearly in a swap but LND is temporarily not connected. Shouldn't we retry?

Reply:
My thinking here is that if lnd is down, we should fail the swap immediately rather than wait in case lnd comes back. The swap could then be retried with a new swap deal. This covers an edge case where lnd goes down in the middle of a swap.

This is the current behavior - there's currently no mechanism to put a swap on hold while we wait to see if lnd comes back online - so I'm just trying to categorize this as an error case where we know that the payment won't go through (because no send payment call has been made, in this case).